api/app/applicationModel/tests.py

```diff
@@ -221,34 +219,34 @@ def test_app_form_is_created(self):
         self.assertIsNotNone(section1_question1['id'])
         self.assertEqual(section1_question1['headline'], APPLICATION_FORM_POST_DATA['sections'][1]['questions'][1]['headline'])
 
-    def test_sections_are_created(self):
-        """
-        Tests that the correct sections are added and can be accessed by the respective app id
-        """
-        pass
 
 
 class ApplicationFormUpdateTest(ApiTestCase):
     """
-    Test that an application form's sections and question are updated
+    Test that an application form's sections and question are updated by an admin with the correct permissions
     """
     def _seed_data_update(self):
-        """
-        Mock data used for test simulation
-        """
-        pass
 
-    def test_section_id_matches_app_id(self):
+        self.event = self.add_event()
+        self.system_admin = self.add_user(is_admin=True)
+        self.event_admin = self.add_user(email='user2@user.com')
+        self.event.add_event_role('admin', self.event_admin.id)
+        db.session.commit()
+
+    def test_event_admin_permission(self):
         """
-        Confirm the sections to be updated belong to the respective application form
+        Tests that the application form is being created by a user with the correct permissions
         """
-        pass
+        self._seed_data_create()
+
+        response = self.app.post(
+            '/api/v1/application-form',
+            # data=json.dumps(APPLICATION_FORM_POST_DATA),
```

Comment:
Will the test for the put method need a new example json?

Reply:
Yeah it would be best to create a new example JSON with the ID populated from the application form and then some sections with IDs and some without, some questions with IDs and some without so you can test the "existing stuff gets updated" and "new stuff gets created" logic.

Reply:
Thank you ! I realised I don't fully understand what it means for an ID to be populated when it comes to looking at the JSON - I understand the concept, but not sure how to create a section  / question without an ID :)